### DISCUSSION TITLE
blocking ads error

### DISCUSSION THREAD

**Discussion by LexterS999** [0xERR0R/blocky#239] [Q&A] [answered]
I have a list of filter's inside my config, but none of them using in Blocky. He is only using what I paste in blacklist.txt

On previous version used the same config and everything was fine. Thought that maybe problem in caching. But the same issue with totally turned off caching feature.

**Comment by 0xERR0R** [comment]
Can you share your config please? Do you use docker or plain binary?

**Comment by LexterS999** [comment]
Binary with nohup.
[config.txt](https://github.com/0xERR0R/blocky/files/6910816/config.txt)

**Comment by 0xERR0R** [comment]
Thx, I'll try it later

**Comment by 0xERR0R** [comment]
Just tested you config, looks good: If I try for example ad.facebook.com (which is on https://raw.githubusercontent.com/DataMaster-2501/DataMaster-Android-AdBlock-Hosts/master/hosts), i get 0.0.0.0:

I changed port to 55555 for testing
```
dig @127.0.0.1 -p 55555 ad.facebook.com 
; <<>> DiG 9.16.18 <<>> @127.0.0.1 -p 55555 ad.facebook.com
; (1 server found)
;; global options: +cmd
;; Got answer:
;; ->>HEADER<<- opcode: QUERY, status: NOERROR, id: 35725
;; flags: qr rd ra; QUERY: 1, ANSWER: 1, AUTHORITY: 0, ADDITIONAL: 0

;; QUESTION SECTION:
;ad.facebook.com.               IN      A

;; ANSWER SECTION:
ad.facebook.com.        21600   IN      A       0.0.0.0

;; Query time: 0 msec
;; SERVER: 127.0.0.1#55555(127.0.0.1)
;; WHEN: Sat Jul 31 22:50:54 CEST 2021
;; MSG SIZE  rcvd: 49
```

**Comment by LexterS999** [comment, marked as answer]
Same for me. It was a problem with nohup. Not saving new config till I delete old one from server and only then it's apply new config. Now I turned it as systemd service and all working fine. Thanks.
